solution_runner/commands/solution_template.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+
+def get_answer(input_text: str):
```

Comment:
Let's try to call the solution code without `subprocess`, to avoid having to write the boilerplate "main function" code. I'll be happy to help.